### PR TITLE
Initial import of harvest docs from azaroth42

### PR DESCRIPTION
**Pull request by azaroth42**
Import revised documentation under harvest and notification.

### CODE REVIEW

File: source/api/notification/0.1/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+ActivityPub is a method for transferring the notifications and subscribing, but does not follow REST patterns. Instead it uses side effects associated with particular activities. For example, the side effect of recieving a "Follow" activity is to add the sender to the "followers" of the resource. A REST based approach would be to create the subscription resource within the followers container.
+
+A more LOD / LDP / LDN appropriate pattern would be to use a REST pattern with Containers, that allow subscription to the inbox, manage filtering and the callback endpoint.  One way to do this could be as [described here](https://docs.google.com/document/d/1JsQS1LVFt8wuJSYo_XsOzyP28pS8hfSLEhAC3BFuN6o/edit).
+
+For the purposes of IIIF discovery, the above decision is firmly in the Notification section of the charter and does not impact the publication of the Activitystreams Collections as resources.
```
Comment (zimeon) [inline source/api/notification/0.1/index.md]: Acitvitystreams -> ActivityStreams

File: source/api/harvest/0.1/index.md [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+* `Update`: Any change to the resource.  In a system that does not distinguish creation from modification, then all changes MAY be `Update`s. 
+* `Delete`: The deletion of the resource, or its de-publication from the web. 
+
+Additional activities which institutions may also find helpful are listed [in this annex](activities/).
+
+__Usage:__  Record each time the list is crawled. Start from the `last` page and work backwards through the list until a datestamp before the previous time a crawl occurred is encountered. Only process the most recent change per resource, including deleting resources.
+
+```json-doc
+{
+	"@context": [
```
Comment (zimeon) [inline source/api/harvest/0.1/index.md]: In other specs we have 2-space indents in our JSON docs, should we keep with that?

File: source/api/harvest/0.1/activities.md [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+	],
+	"target": {
+		"id": "https://example.org/iiif/museum/1726/manifest.json",
+		"type": "Manifest"
+	},
+	"endTime": "2017-09-19T20:00:00Z"
+}
+```
+
+#### Split
```
Comment (zimeon) [inline source/api/harvest/0.1/activities.md]: Is there really any good reason to have split? If the expected behavior is to reharvest both of the new resources then that is the same as delete and 2 creates. 

Saying that the UI might want to group things seems wrong in that one is then saying that the update system is conveying presentation semantics that aren't present in the current state of the resources -- so the grouping would then vanish at some later time?

File: source/api/harvest/0.1/activities.md [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+	},
+	"target": {
+		"id": "https://example.org/iiif/publications/102314/manifest.json",
+		"type": "Manifest"
+	},
+	"endTime": "2017-09-19T20:00:00Z"
+}
+```
+
+#### Merge
```
Comment (zimeon) [inline source/api/harvest/0.1/activities.md]: I don't think we should include Merge without a use case, this seems to violate the "As Simple as Possible, and No Simpler" principle

File: source/api/notification/0.1/index.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# IIIF Notifications
+
+This information is extracted from the Crawling work, as it applies to notifications using the ActivityStreams pattern.
```
Comment (zimeon) [inline source/api/notification/0.1/index.md]: Add link for "Crawling work" and "ActivityStreams" given first mention in the doc?

**Review by zimeon — CHANGES_REQUESTED** [review]

File: source/api/harvest/0.1/activities.md (hunk 4 above)
Comment (azaroth42) [reply source/api/harvest/0.1/activities.md]: How about adding a :warning: warning block to Copy and Merge that they need use cases?  If we take them out completely, they'll get lost.

Comment (zimeon) [reply source/api/harvest/0.1/activities.md]: I guess that is a reasonable compromise for an in-process document

Comment (azaroth42) [reply source/api/harvest/0.1/activities.md]: Done.

File: source/api/harvest/0.1/activities.md (hunk 3 above)
Comment (azaroth42) [reply source/api/harvest/0.1/activities.md]: Agree the UI is not a good consideration.  Added the same warning as for Merge and removed the UI note.

File: source/api/harvest/0.1/index.md (hunk 2 above)
Comment (azaroth42) [reply source/api/harvest/0.1/index.md]: Yes. Can we do this in a second round?

File: source/api/notification/0.1/index.md (hunk 5 above)
Comment (azaroth42) [reply source/api/notification/0.1/index.md]: Done

File: source/api/notification/0.1/index.md (hunk 1 above)
Comment (azaroth42) [reply source/api/notification/0.1/index.md]: Done

File: source/api/harvest/0.1/index.md (hunk 2 above)
Comment (azaroth42) [reply source/api/harvest/0.1/index.md]: Second thoughts ... done :)

**Review by zimeon — APPROVED** [review]

**Review by mikeapp — APPROVED** [review]

**Pull request merged**